src/StreamReader.php

```diff
@@ -22,6 +22,8 @@ class StreamReader implements IteratorAggregate
     /** @var resource */
     private $stream;
 
+    private ?Header $globalPaxHeader = null;
```

Comment:
This variable is looks never filled. This variable is too general stub. Please remove it or implement fiiliing.

Reply:
[095a6ba](https://github.com/jakubboucek/php-tar-stream-reader/pull/5/commits/095a6ba0735929f7fb20c6ac672bccee11ecb1b5)